src/main/java/com/parsing/rozetka/RozetkaParser.java

```diff
@@ -0,0 +1,89 @@
+package com.parsing.rozetka;
+
+import com.parsing.model.LotResult;
+import com.parsing.model.RozetkaParsingReport;
+import com.parsing.model.LaptopItem;
+import com.parsing.model.Status;
+import com.parsing.repository.LotResultRepository;
+import com.parsing.repository.RozetkaParsingResultRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+import org.openqa.selenium.WebDriver;
+import org.springframework.stereotype.Component;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+@Component
+public class RozetkaParser {
+
+    private static final String ROZETKA_URL = "https://rozetka.com.ua/";
+    private static final String SEARCH = "search/";
+    private static final String SEARCH_PARAM = "?text=";
+    private static final String LAPTOP_SECTION_ID = "&section_id=80004";
+
+    private final LotResultRepository lotResultRepository;
+    private final RozetkaParsingResultRepository rozetkaParsingResultRepository;
+    private final WebDriver driver;
+
+    public List<RozetkaParsingReport> parsingByLotResult(LotResult lotResult) {
```

Comment:
methods names start with a verb, "parsing" is a noun which is appropriate for a class name but not for a method